### ISSUE TITLE
Ironing board

### ISSUE THREAD

**Issue by iteles** [dwyl/home#165]
I'm full of exciting issues lately 😆

**Comment by iteles** [comment]
I have done some research and seems like the best place to get this would be just straight up in one of the bigger supermarkets. 

Continente has the better choice:
![image](https://user-images.githubusercontent.com/4185328/101072172-8b654080-3595-11eb-9034-e772f2a94184.png)


Of these, the only one that actually has enough information for us to confirm that it is height adjustable and has a cover rather than a non-washable plate is this one:
![image](https://user-images.githubusercontent.com/4185328/101072111-738dbc80-3595-11eb-98ae-0882d01c0353.png)


Although I assume the 'premium' one is also height adjustable, but looks extra bulky to me.
The different is just 5cm in length and barely anything in width (the measurements in the description above say the one I'm proposing is actually 45cm wide and not 41cm).

Happy for me to just get the white one?

**Comment by nelsonic** [comment]
@iteles Yeah, happy for you to get whichever you feel meets our needs. :+1: (_thanks!_)

**Comment by iteles** [comment]
Done. We have to pick it up at Continente between 10h00 and 13h00 tomorrow 👍 

If possible, we probably want to be there at 10h00 on the dot because you saw the queue at the customer services section when we were there last weekend! 😬
